.gitignore

```diff
@@ -108,3 +108,6 @@ db_m
 
 # TernJS port file
 .tern-port
+
+# Others
```

Comment:
nit: I would comment this as Authentication or Credentials, Others seems pretty vague